MQTicketService/TicketGenerator/com/ibm/mq/badge/EnvSetter.java

```diff
@@ -182,10 +184,11 @@ private void setJMSProperties(JmsConnectionFactory cf) {
             cf.setIntProperty(WMQConstants.WMQ_PORT, PORT);
             // cf.setIntProperty(WMQConstants.WMQ_CLIENT_RECONNECT_OPTIONS, WMQConstants.WMQ_CLIENT_RECONNECT);
             cf.setStringProperty(WMQConstants.WMQ_CHANNEL, CHANNEL);
-            cf.setIntProperty(WMQConstants.WMQ_CONNECTION_MODE, WMQConstants.WMQ_CM_BINDINGS);
-            // cf.setBooleanProperty(WMQConstants.USER_AUTHENTICATION_MQCSP, true);
-            // cf.setStringProperty(WMQConstants.USERID, USER);
-            // cf.setStringProperty(WMQConstants.PASSWORD, PASSWORD);
+            // cf.setIntProperty(WMQConstants.WMQ_CONNECTION_MODE, WMQConstants.WMQ_CM_BINDINGS);
+            cf.setIntProperty(WMQConstants.WMQ_CONNECTION_MODE, WMQConstants.WMQ_CM_CLIENT);
+            cf.setBooleanProperty(WMQConstants.USER_AUTHENTICATION_MQCSP, true);
```

Comment:
Clean up the indentation in the method setJMSProperties

Reply:
Removed unnecessary indentations.